procursus-deploy-linux-macos.sh

```diff
@@ -100,17 +101,17 @@ EOF
 
 echo "(1) Downloading resources..."
 IPROXY=$(iproxy 28605 44 >/dev/null 2>&1 & echo $!)
-curl -sLOOOOO https://github.com/coolstar/Odyssey-bootstrap/raw/master/bootstrap_1500.tar.gz \
-	https://github.com/coolstar/Odyssey-bootstrap/raw/master/bootstrap_1600.tar.gz \
-	https://github.com/coolstar/Odyssey-bootstrap/raw/master/bootstrap_1700.tar.gz \
-	https://github.com/coolstar/Odyssey-bootstrap/raw/master/org.coolstar.sileo_2.2.3_iphoneos-arm.deb \
+curl -sLOOOOO https://github.com/coolstar/Odyssey-bootstrap/raw/master/bootstrap-1500.tar.gz \
```

Comment:
This is nitpick-y, but can you change the bootstrap file names back to underscores instead of dashes?